### DISCUSSION TITLE
Bots detection

### DISCUSSION THREAD

**Discussion by Ziedelth** [vinceanalytics/vince#17] [General]
Hello,
I hope you are well.

I reinstalled vince locally (with Docker) following the last updates. For the moment, everything is working properly, but I have the bots coming up in the vince detections. Particularly Google bots when in mobile mode.

Will this be fixed in future updates?

PS: Too bad the issues are disabled, I get quite a few 404 and 500 errors when browsing vince...

**Comment by gernest** [comment]
Hi, 

Yes,  I am cleaning up user agents implementation, eventually I will enable bots detection again.

> PS: Too bad the issues are disabled, I get quite a few 404 and 500 errors when browsing vince.

I am a solo founder on a very early stage product. Vince is pre alpha software. It will take sometime  until I make an official release. Issues have been overwhelming for me.

I am happy to discuss things, but I don't have the bandwidth to resolve issues at the moment as things are still changing.

**Reply by Ziedelth** [reply]
Hi, there,
Sorry for the late reply.

I completely understand that the software is in pre-alpha, but you've answered my question about bots.

I'd be happy to help, but I don't know anything about Go... but good luck!

To come back to the bots, it seems that there are also Threads bots that come to the site when you publish a link, but they seem to use a browser, with no predefined user agent...

**Reply by gernest** [reply]
> I'd be happy to help, but I don't know anything about Go... but good luck!

Thanks

> To come back to the bots, it seems that there are also Threads bots that come to the site when you publish a link, but they seem to use a browser, with no predefined user agent...

Please check this file https://github.com/matomo-org/device-detector/blob/master/regexes/bots.yml to see if the bot user agent is there. And if not submit a patch there to make sure it is properly classified.

**Comment by gernest** [comment]
@Ziedelth I have just made an official release, with bot detection support.

You can see the demo on the README. 

Thanks for being patient with me.
